sites/public/src/components/shared/MapControl.module.scss

```diff
@@ -0,0 +1,33 @@
+
+.map-control {
+  --control-border-color: var(--bloom-color-blue-900);
```

Comment:
nit: --bloom-color-blue-900 is the primary color so it might be better to set this to --bloom-color-primary so it changes for free if we decide to change the shade of the primary color.